packages/subscription-manager/src/components/BlockEmailsSetting/block-emails-setting.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-restricted-imports */
+/**
+ * External dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import { ChangeEventHandler } from 'react';
+import FormInputCheckbox from 'calypso/components/forms/form-checkbox';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormLabel from 'calypso/components/forms/form-label';
+import './style.scss';
+
+export type BlockEmailsSettingProps = {
+	value: boolean;
+	onChange: ChangeEventHandler< HTMLInputElement >;
+};
+
+const BlockEmailsSetting = ( { value, onChange }: BlockEmailsSettingProps ) => {
+	const translate = useTranslate();
+
+	return (
+		<FormFieldset className="block-emails-setting">
+			<span className="title">Block emails</span>
+			<FormLabel>
+				<FormInputCheckbox name="block_emails_setting" checked={ value } onChange={ onChange } />
```

Comment:
Not sure if we will end up using the name of the input, but to match our API the input should have a name of "blocked"

Reply:
I'll change it 👍